.github/workflows/CI.yaml

```diff
@@ -178,3 +178,19 @@ jobs:
 
     - name: conformance
       run: TEST=true PYTHONPATH=. pytest test/registry/conformance_tests.py -s -vv
+
+  commit-check:
+    name: Commit Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: commit check
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: |
+            ^(.*):\s*(.*)\s\(PROJQUAY-[0-9]+\).*$
```

Comment:
Add a new line before the end: `^(.*):\s*(.*)\s(\(PROJQUAY-[0-9]+\))\n.*$`

Reply:
i had that originally but it was failing my one line commit message, are you saying it's required for the actions?

Reply:
As it is now `scope: somemessage header (PROJQUAY-1234 )and some other stuff` would be valid.
New line makes sure the last part of the header is the ticket number.

Reply:
updated but you can see that it is now failing my single line commit

Reply:
Right, you should add a new line to your message.
It is generally good practice to have a commit message anyways.

Reply:
changed to accept single or multi line